lib/services/autofs.pm

```diff
@@ -0,0 +1,100 @@
+# SUSE's openQA tests
```

Comment:
Since we have autofs_utils.pm already, so this file's content should move to autofs_utils.pm.